examples/huggingface/pytorch/code-generation/quantization/requirements.txt

```diff
@@ -12,4 +12,4 @@ neural-compressor
 intel_extension_for_pytorch==2.2.0
 optimum-intel
 auto-round
-git+https://github.com/bigcode-project/bigcode-evaluation-harness@00967d12093ef614de7bdad0772aed8e4118f1fd
+git+https://github.com/bigcode-project/bigcode-evaluation-harness@094c7cc197d13a53c19303865e2056f1c7488ac1
```

Comment:
why for quantization we still need commit id  and  for inference we don't need it

Reply:
it is bigcode-evaluation-harness, not lm-eval and used only for code-generation, it doesn't have pip release version, only support source install.
https://github.com/bigcode-project/bigcode-evaluation-harness